### PR TITLE
Removes comments for time argument for Mixer Fade functions

### PR DESCRIPTION
**Pull request by cugone**
Removes low-value comments for time argument for Fade functions due to explicit type for milliseconds.

### CODE REVIEW

**Comment by cugone** [comment]
@DanRStevens This shouldn't conflict with #951

**Comment by DanRStevens** [comment]
Could you please either rebase this branch on master or merge master into this branch? I'm seeing a test failure, though I think it's the problem that was just fixed by merging #951.

File: NAS2D/Mixer/Mixer.h [hunk 1]
```diff
@@ -56,15 +56,9 @@ namespace NAS2D
 		 *
 		 * \param music Reference to a Music Resource.
 		 * \param loops Repeat count. -1 for continuous loop.
-		 * \param time Length of the fade in milliseconds. Default is 500.
```
Comment (DanRStevens) [inline NAS2D/Mixer/Mixer.h:59]: I'm a little bit concerned that documenting some but not all parameters could trigger a warning/error from some of the documentation checks. Would be nice to get all CI builds to run and see if they complain.

Comment (cugone) [reply NAS2D/Mixer/Mixer.h:59]: Would removing the entire comment block be better?

Comment (DanRStevens) [reply NAS2D/Mixer/Mixer.h:59]: Possibly. I feel a little on the fence on this one.

I think maybe the parameter should be renamed to something like `fadeTime`. Then I'd probably be comfortable removing the documentation.

**Review by DanRStevens — APPROVED** [review]

**Pull request merged**